newdle/api.py

```diff
@@ -387,7 +404,7 @@ def create_participant(code):
             name=name, email=g.user['email'], auth_uid=g.user['uid']
         )
         newdle.participants.add(participant)
-        db.session.commit()
+        db.session.commit()        
```

Comment:
I'm very surprised this wasn't removed by `black` during the pre-commit autoformatting... In any case, you may want to configure your editor to remove trailing whitespace since it's pretty much never wanted.